src/components/button.js

```diff
@@ -15,15 +25,24 @@ AFRAME.registerComponent("button", {
 
     el.setAttribute("material", {
       shader: "buttonShader",
-      borderRadius: 0.28,
+      colorPrimary: colorPrimary,
+      colorSecondary: colorSecondary,
+      borderRadius: borderRadius,
       opacity: 1,
       borderWidth: 0.02,
     });
 
     el.setAttribute("text", {
       value: value,
       align: "center",
-      width: width,
+      width: widthText,
+    });
+    el.setAttribute("class", "clickable");
+    el.addEventListener("mouseenter", function () {
+      el.setAttribute("geometry", { width: 1.6, height: 0.6 });
```

Comment:
Would these hard-coded values work with any other `borderRadius`?

Reply:
They should but I'm gonna change that so that they aren't hard coded.